javascript/javascript3/week3/shopping_cart/main.js

```diff
@@ -0,0 +1,93 @@
+const userSection = document.querySelector('#user');
+const productsList = document.querySelector('#productsList');
+
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+
+    convertToCurrency(currency) {
+       const currencyRate = {
+            'krone' : 1,
+            'dollar' : 0.16,
+            'euro' : 0.13,
+            'hryvnia' : 4.66
+        }
+        return this.price * currencyRate[currency];
```

Comment:
hard coding currency rates is ok for the exercise, but hopefully you would use an async call to some API in a "real" situation